main/modules/files.js

```diff
@@ -420,6 +420,10 @@ function createRTFConversionFunction(sender) {
 }
 
 function createFromScrivener(importedPath, sender, isLoggedIntoPro) {
+  if (importedPath.error) {
+    log.error(importedPath.error)
+    return sender.send('finish-creating-local-scrivener-imported-file')
```

Comment:
See my previous comment.  We need to let the user know that something went wrong.